tests/test_deployment.py

```diff
@@ -910,6 +910,15 @@ def test_failure(self, debugger):
 @pytest.mark.slow
 @pytest.mark.docker
 class TestDockerServer(object):
+    @pytest.fixture(autouse=True)
```

Comment:
I've forgotten the scope of a fixture-as-method; is it implicitly given `class` scope? Or `function` scope limited to test methods within the class? (Just want to be sure we're not accidentally executing this for every test!)

Reply:
It's limited to test methods defined in the class.

I checked this by placing a breakpoint in the fixture function and invoking
```
pytest  --runslow --chrome-headless -s tests/test_deployment.py::TestSetupExperiment tests/test_deployment.py::TestDockerServer
```

The breakpoint was traversed twice while 18 tests were run.